src/dndbeyond/base/utils.js

```diff
@@ -223,6 +223,19 @@ async function buildAttackRoll(character, attack_source, name, description, prop
                 crit_damages[0] = damagesToCrits(character, ["8d12"])[0];
             if (roll_properties.name === "Jim’s Magic Missile")
                 crit_damages[0] = damagesToCrits(character, ["3d4"])[0];
+            const matchViciousWeapon = description.match(/(When you roll a 20 on your attack roll with this magic weapon, the target takes an extra )(\d+)( damage of the weapon’s type.)/);
```

Comment:
You don't need to use capture groups (parenthesis) around the first/last part of the text. You just need one value (the damage amount) so you only need one capture group. 
```js
const matchViciousWeapon = description.match(/When you roll a 20 on your attack roll with this magic weapon, the target takes an extra (\d+) damage of the weapon’s type./);
```
You can then refer to it with `matchViciousWeapon[1]` instead of `matchViciousWeapon[2]`